styles/searxng/catppuccin.user.css

```diff
@@ -17,120 +17,9 @@
-domain("search.affusio.com"),
-domain("search.bus-hit.me"),
-domain("search.callmeecho.dev"),
-domain("search.chemicals-in-the-water.eu"),
-domain("search.cronobox.one"),
-domain("search.disroot.org"),
-domain("search.ethibox.fr"),
-domain("search.gcomm.ch"),
-domain("search.kiwitalk.de"),
-domain("search.mdosch.de"),
-domain("search.mpx.wtf"),
-domain("search.neet.works"),
-domain("search.ononoki.org"),
-domain("search.privacyguides.net"),
-domain("search.projectsegfau.lt"),
-domain("search.rabbit-company.com"),
-domain("search.rhscze.cf"),
-domain("search.rowie.at"),
-domain("search.sapti.me"),
-domain("search.serginho.dev"),
-domain("search.smnz.de"),
-domain("search.snopyta.org"),
-domain("search.stinpriza.org"),
-domain("search.teamriverbubbles.com"),
-domain("search.trom.tf"),
-domain("search.unlocked.link"),
-domain("search.uspersec.com"),
-domain("search.vidhukant.xyz"),
-domain("search.zzls.xyz"),
-domain("searx.baczek.me"),
-domain("searx.be"),
-domain("searx.becomesovran.com"),
-domain("searx.bissisoft.com"),
-domain("searx.chocoflan.net"),
-domain("searx.divided-by-zero.eu"),
-domain("searx.dresden.network"),
-domain("searx.ebnar.xyz"),
-domain("searx.ericaftereric.top"),
-domain("searx.esmailelbob.xyz"),
-domain("searx.fi"),
-domain("searx.fmac.xyz"),
-domain("searx.fossencdi.org"),
-domain("searx.gnous.eu"),
-domain("searx.gnu.style"),
-domain("searx.kujonello.cf"),
-domain("searx.mastodontech.de"),
-domain("searx.mha.fi"),
-domain("searx.mistli.net"),
-domain("searx.mxchange.org"),
-domain("searx.nakhan.net"),
-domain("searx.namejeff.xyz"),
-domain("searx.netzspielplatz.de"),
-domain("searx.nixnet.services"),
-domain("searx.org"),
-domain("searx.orion-hub.fr"),
-domain("searx.priv.pw"),
-domain("searx.prvcy.eu"),
-domain("searx.rimkus.it"),
-domain("searx.roflcopter.fr"),
-domain("searx.ru"),
-domain("searx.run"),
-domain("searx.semipvt.com"),
-domain("searx.sethforprivacy.com"),
-domain("searx.sev.monster"),
-url-prefix("https://searx.slipfox.xyz/searx/"),
-domain("searx.sp-codes.de"),
-domain("searx.stuehieyr.com"),
-domain("searx.tiekoetter.com"),
-domain("searx.tuxcloud.net"),
-domain("searx.tyil.nl"),
-domain("searx.vanwa.tech"),
-domain("searx.webheberg.info"),
-domain("searx.xyz"),
-domain("searx.zapashcanon.fr"),
-domain("searx.zcyph.cc"),
-domain("searxng.ir"),
-domain("searxng.nicfab.eu"),
-domain("searxng.zackptg5.com"),
-domain("serx.ml"),
-domain("sh0.it"),
-domain("soek.allesbeste.com"),
-domain("spot.ecloud.global"),
-domain("srx.cosmohub.io"),
-domain("searx.oakleycord.dev"),
-domain("suche.dasnetzundich.de"),
-domain("suche.tromdienste.de"),
-domain("suche.uferwerk.org"),
-domain("swag.pw"),
-domain("sx.catgirl.cloud"),
-url-prefix("https://timdor.noip.me/searx/"),
-url-prefix("https://trydex.tk/searxng/"),
-domain("www.gruble.de"),
-domain("www.webrats.xyz"),
-domain("xcxc.ml"),
-domain("xo.wtf") {
+@-moz-document
+domain("https://search.im-in.space/"), 
+domain("https://search.projectsegfau.lt/") {
```

Comment:
Possibly stupid question but why these two specifically?

Reply:
> Possibly stupid question but why these two specifically?

i just googled the instances list and picked these two

Reply:
also they're pretty stable too

Reply:
These two weren't even on the original list though?

Reply:
Do you use SearXNG?

Reply:
a bit, yeah

Reply:
Maybe just take the top three or something? Also leave a comment above the `@-moz-document` detailing where these are from would be great - `/* Domains picked from https://searx.space/. */

Reply:
that sounds like a great idea

Reply:
@uncenter i did all of your suggestions